Add error hook (#68)
Actyx/machines: machine-runner/README.md

## Features

### Observe Changes and Errors

An alternative use case of a machine runner is to listen to its events.

> `next` event emits states whenever a new state is calculated.

Reviewer: ```suggestion
The `next` event emits states whenever a new state is calculated.
```

> Calling `destroy` is required when the machine is not used to sever subscription from Actyx.

Reviewer: ```suggestion
When not using the machine, calling `destroy` is required to close the connection to Actyx.
```

```typescript
const warehouse = createMachineRunner(actyx, tags, InitialWarehouse, { id: '4711' })

warehouse.events.on('next', (state) => {
  if (state.is(InitialWarehouse)) {
    // ...
  }
})

await untilWareHouseIsNotUsedAnymore()

warehouse.destroy()
```

> `error` event can be used to capture errors that happens from machine-runner.

Reviewer: ```suggestion
`error` event can be used to capture errors from machine-runner.
```

  })
  ```

### Global Event Emitter

Some global event emitters are provided.

> This event emitters will emit events from all machine runners in the same process.

Reviewer: ```suggestion
These event emitters will emit events from all machine runners in the same process.
```
